tests/config/test_themes.py

```diff
@@ -213,3 +215,25 @@ def test_add_pygments_style(
     # Check for overrides(k,sd) and inheriting styles (kr)
     for style in expected_styles:
         assert style in urwid_theme
+
+
+# Validate 16-color-codes
+@pytest.mark.parametrize(
+    "color_depth, theme_name",
+    [
+        (16, "zt_dark"),
+        (16, "gruvbox_dark"),
+        (16, "gruvbox_light"),
+        (16, "zt_light"),
+        (16, "zt_blue"),
+    ],
+)
+def test_validate_colors(theme_name: str, color_depth: int) -> None:
+    class Color(Enum):
+        WHITE = "whit  #256  #24"
+
+    theme = THEMES[theme_name]
+    theme.Color = Color
+
+    with pytest.raises(Check16ColorCode):
+        validate_colors(theme_name, 16)
```

Comment:
This is a good basic test, but it would be useful to also confirm what text is in the exception (particularly for different cases).